src/System.Diagnostics.Contracts/tests/BasicContractsTests.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.Contracts;
+using Xunit;
+
+namespace System.Diagnostics.Contracts.Tests
+{
```

Comment:
It's not very much, this is what we have for our existing assets, I'd like to supplement them.

@stephentoub who had the same question - this commit from dotnet-bot has the initial ported assets which I'd like to supplement post //BUILD.  We should also look at running the rewriter in CI as well.